openassessment/xblock/team_mixin.py

```diff
@@ -109,11 +112,23 @@ def get_team_info(self):
         if self.in_studio_preview:
             return self.STAFF_OR_PREVIEW_INFO
         elif self.has_team():
+            student_item_dict = self.get_student_item_dict()
```

Comment:
This should be moved to submission_mixin, to maybe be refactored into a get_team_context method.

Reply:
I would like to hear more on your thoughts as to why. Do you mean the entire new block should be in the submission_mixin?